static/styles/settings.scss

```diff
@@ -544,7 +544,7 @@ input[type=checkbox] {
     }
```

Comment:
@sahil839 can you add `setting:` before the commit text and elaborate the commit a little.

Reply:
Added the explanation.